### DISCUSSION TITLE
Update Azuracast

### DISCUSSION THREAD

**Discussion by heyDaino** [AzuraCast/AzuraCast#5319] [Q&A]
Hello.

Could you help me to update Azuracast?
I'm sure it's an easily solved problem by granting permissions to docker, but I can't find any commands to share the privileges. Unfortunately I'm a nub :)

![Screenshot_1](https://user-images.githubusercontent.com/70600193/165781766-350ea20d-ff41-4d5e-8ac2-0e0745c3e84e.jpg)

**Comment by Vaalyn** [comment]
Try running the commands with `sudo` like this:

```
sudo ./docker.sh update-self
sudo ./docker.sh update
```

**Reply by heyDaino** [reply]
Thank you <3
